rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.77"
+channel = "nightly"
```

Comment:
Is this change required?

Reply:
Yes, this is required by AVX512.  And that's why I try to merge it into powdr-labs:avx

Reply:
There is another options:  creating a new rust-toolchain in backend,  I am not sure which one is better.